config/config.go

```diff
@@ -162,6 +163,12 @@ type BootstrapConf struct {
 	File string
 }
 
+type PrometheusConf struct {
+	Enabled bool
+	Network NetworkConf
+	TLS     TLSConf
+}
+
```

Comment:
looks good. please document it and add some tests to config_test.go and items to the testdata folder that would cover the correct reading of a config with this item.

Reply:
added